test/manipulation.test.js

```diff
@@ -659,10 +659,60 @@ describe('manipulation', function() {
   });
 
   describe('updateOrCreate', function() {
+    var db = getSchema();
+    var Todo;
+
+    before(function setUp(done) {
+      Todo = db.define('Todo', {
+        content: String,
+      });
+      db.automigrate('Todo', done);
```

Comment:
I've also been thinking, we should implement some code that automigrates particular models if it doesn't exist. Seems like something the system can do behind the scenes without manually forcing users to understand (I've hit this issue a few times in community). Something like:

```
find -> check if sql connector -> check if table exists -> if not, try to automigrate -> if no model available for automigration return error, otherwise automigrate the missing table
```

Of course we'd have to flush it out a bit more, but the user experience should be like the mongo one (don't have to worry about the automigration of tables that don't exist already -- and have a flag to optionally disable this feature).